src/test_helpers/tests/mock_stream_evaluator.h

```diff
@@ -0,0 +1,15 @@
+#include "gmock/gmock.h"
+#include "../stream_evaluator_i.h"
+
+namespace btest {
+
+class MockStreamEvaluator : public StreamEvaluatorI {
+ public:
```

Comment:
I would still strongly against using one space for specifiers as almost no IDE would be able to be tuned to do it. So if it's doable to change it easily in emacs setting, I would suggest change it to using zero indent for specifiers.